Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -1261,9 +1261,39 @@ def test_module(params: dict) -> str:
     return 'ok'
 
 
+def edit_server_info(server_info: dict) -> dict:
+    """Edits the server info dictionary if the server version >= 8.0.0
+
+    Args:
+        server_info (dict): The server info
+    """
+    if is_demisto_version_ge('8.0.0'):
+        altered_api_roots = []
+        for api_root in server_info['api_roots']:
```

Comment:
In case api_roots is none 
```suggestion
        for api_root in server_info.get('api_roots') or []:
```